pkg/blockchain/config.go

```diff
@@ -14,25 +14,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package apiserver
+package blockchain
 
-import (
-	"bytes"
-	"net/http/httptest"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/mock"
+const (
+	// BlockchainConfigType is the type of blockchain used for this blockchain plugin
+	BlockchainConfigType = "type"
```

Comment:
Really feels like these constants should be global across all plugins (maybe in `core` or `coreconfig`), instead of declaring them in each plugin package.